css/properties/transition-property.json

```diff
@@ -0,0 +1,162 @@
+{
+  "css": {
+    "properties": {
```

Comment:
The comments about transition-delay.json are also valid here. Please apply them here too.